test/store/update.unit.ts

```diff
@@ -19,4 +19,44 @@ describe('store | update', () => {
       expect(updateOne(store, 'b', update)).to.eql({ allKeys: ['a'], byKey: { a: value, b: update } });
     });
   });
+
+  describe('updateMany()', () => {
+    const store = normalize([
+      { id: 'a', foo: false },
+      { id: 'b', foo: true },
+    ]);
+
+    it('replaces an existing values', () => {
+      const updateA = { id: 'a', foo: true };
+      const updateB = { id: 'b', foo: false };
+
+      expect(updateMany(store, [updateA, updateB])).to.eql({ allKeys: ['a', 'b'], byKey: { a: updateA, b: updateB } });
+    });
+
+    it('adds a new values when key is not recognized', () => {
+      const update = { id: 'c', foo: true };
+
+      expect(updateMany(store, [update])).to.eql({ allKeys: ['a', 'b'], byKey: { ...store.byKey, c: update } });
+    });
```

Comment:
should this be a behaviour we allow? `c` does not get added to `allKeys`

Reply:
yeah, I've copied this behavior from the `updateOne` method, and looks like we did it intentionally, cause `updateOne` has tests for this case